includes/definitions/discovery/linux.yaml

```diff
@@ -35,27 +35,27 @@ modules:
                     descr: 'Virtual disks'
                 -
                     oid: adapterPropertiesTable
-                    value: pdDiskPredFailureCount
-                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.1.2.1.19.{{ $index }}'
+                    value: vdOfflineCount
+                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.1.2.1.20.{{ $index }}'
                     index: 'vdfail.{{ $index }}'
                     group: 'RAID Controller {{ $adapterID-APT }}'
-                    warn_limit: 1
+                    high_limit: 0.1
```

Comment:
Why did you set these to 0.1?  It should probably be an integer.

Reply:
It needs to be 0, but that evaluates to false, making it get set to null in the config and not alerting.  If it is set to 1 it won't trigger until more than 1 disk fails, which is bad.  The database value is a float, so 0.1 fixes the issue of not alerting.

A better fix would be to know where these options are translated from YAML to the database and differentiate between 0 and not set.  This would also require knowing where the limits are used for alerting and making sure that this code also differentiates between 0 and not set.  It may also require checking the config web page to do the same.

In this case, this patch would still be relevant, but all the 0.1 values would become 0.

Reply:
I can't reproduce that.  0 values are stored properly.

Reply:
Also, wouldn't a value of 0 always be in alarm?

Reply:
See screenshot below - the value of 0 with high limit of 0 does not result in an alarm.